packages/fxa-settings/src/components/Settings/DataCollection/en.ftl

```diff
@@ -1,9 +1,9 @@
 ## Data collection section
 
 dc-heading = Data Collection and Use
-dc-subheader = Help improve { -product-firefox-accounts }
```

Comment:
@bcolsson I've renamed the strings that use the `-product-firefox-accounts` branding term, as this changes from `Firefox Accounts` (translating Accounts was optional) to `Firefox accounts`(translating accounts recommended). I assumed these strings should all be reviewed? 

Thankfully only 6 strings used this term, and Firefox Accounts/Firefox accounts was the only discrepancy with the terms userd in fxa-auth-server.

Reply:
So the challenge here is, we are replacing a term with another, but the IDs are the same but the content differs.
When we change these 6 IDs, localizers will not see any change in Pontoon from the previous translation. This is because the fact we swapped out the term behind the scenes is invisible to them.

Instead of triggering unnecessary translations for all locales, it may be better to investigate which locales have different translations for the two terms being swapped and notify them of the change. Doing a very rough comparison between the translations of these terms, the number with differing translations is minimal.

Reply:
Interesting, I didn't know that there was an option to notify specific teams about changes. Is that something that you would handle? And does that mean that it's unnecessary to change the IDs here?

I imagine we'd most like to notify teams that treated "Firefox Accounts" as brand term that shouldn't be localized? (Italian, for example)

Reply:
Yes, as an Admin that is something I do have as an option. If we did this we wouldn't need to change the IDs here.

>I imagine we'd most like to notify teams that treated "Firefox Accounts" as brand term that shouldn't be localized? (Italian, for example)

I was making this suggestion for teams that have different translations between the versions for `{ -product-firefox-accounts }`. Basically any team whose on screen text would appear differently than before.

Reply:
Sounds good, I'll revert these ID changes!

Reply:
Noting here: After running a script to compare any strings with shared IDs between `branding.ftl` and `settings.ftl` I've found that not only `-product-firefox-accounts` but also `-product-firefox-account` has some differences in translations between the two files.

Reply:
Thanks for checking on that, @bcolsson! Is any action required on my part to address those differences before merging in this PR?